File: Assignment_41/program5.c
// Write a recursive program which display below pattern.  
//  Input : 5 
//  Output : a b c d e


///////////////////////////////////////////////////////////////////////////////
// File Name   :  program5.c
// Description :  Recursive program which display below pattern.
// Autor       :  Ankita Anil Patil
// Date        :  23/07/2025
///////////////////////////////////////////////////////////////////////////////

#include<stdio.h> 
void Display(int iNo) 
{ 
    static char ch = 'a';
    if(iNo >= 1)
    {
        printf("%c\t",ch);
        ch++;
        Display(--iNo);
        
    }
} 
int main() 
{ 
    int iValue = 0; 

    printf("Enter number : "); 
    scanf("%d",&iValue); 
    Display(iValue); 

    return 0; 
}

File: Assignment_55/program5.java
// Write Java program which accept array from user and display below pattern. 
/*
    Input :     8  6 4 2 4     
    Output :    * * * * * * * * 
                * * * * * * 
                * * * * 
                * * 
                * * * * 
 */

////////////////////////////////////////////////////////////////////////////////
// File Name   :  program5.java
// Description :  Accept array from user and display below pattern       
// Autor       :  Ankita Anil Patil
// Date        :  13/09/2025
///////////////////////////////////////////////////////////////////////////////


import java.util.*;

class MyArray 
{ 
    public void Pattern(int arr[])
    {
        int i = 0, j = 0;
        for(i = 0; i < arr.length;i++)
        {
            for(j = 0; j < arr[i]; j++)
            {
                System.out.print("*\t");
            }
            System.out.println();
        }

    }
        
} 
public class program5
    
{   
    public static void main(String[] args) {
        

    Scanner sobj = new Scanner(System.in);

    System.out.println("Enter size of array");
    int iSize = sobj.nextInt();


    int[] Arr = new int[iSize];
    System.out.println("Enter number one by one that you want to print * on each row  : ");

    int i = 0;
    
    for(i = 0; i < Arr.length; i++)
    { 
       Arr[i] = sobj.nextInt();
    }

    MyArray mobj = new MyArray();
    mobj.Pattern(Arr);

 
}
}
